__tests__/app.test.js

```diff
@@ -110,4 +110,85 @@ describe("app", () => {
         });
     });
   });
+
+  describe("POST /api/reviews/:review_id/comments, adds comment to a review", () => {
```

Comment:
1 more test for where some extra properties are sent in the request body but are ignored.